css/style.css

```diff
@@ -0,0 +1,391 @@
+  top: 0;
+  left: 0;
+  background-color: #fff;
+  margin-bottom: 20%;
+}
+
+.close {
+  width: 50px;
+  height: 50px;
+  margin-top: 20px;
+}
+
+.links {
+  padding-top: 10px;
+  margin: 0 0 10px 20px;
+}
+
+.menu-links {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.menu-ul.active {
+  display: flex;
+}
+
+.pg1-sec {
+  background-image: url("../Images/background.png");
+  height: auto;
+  padding: 80px 50px;
+  width: 100%;
+  padding-top: 210px;
+}
+
+.hello-world {
+  font-size: 25px;
+  font-family: 'Inter', sans-serif;
+  color: #ec5242;
+  margin-bottom: 10px;
+}
+
+.event-name {
+  font-size: 45px;
+  font-family: 'Crete Round', sans-serif;
+  color: #ec5242;
+}
+
+.event-dis {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  border: 6px solid #fff;
+  padding: 20px 20px;
+  margin-top: 20px;
+}
+
+.event-start {
+  display: flex;
+  justify-content: center;
+  font-size: 24px;
+  padding: 20px 20px;
+  margin-top: 20px;
+}
+
+.user-sec {
+  background-image: url("../Images/user-sec.jpg");
+  width: 100%;
+  height: auto;
+  padding: 20px 20px;
+}
+
+.program-h1 {
+  text-align: center;
+  font-size: 26px;
+  color: #fdfefd;
+}
+
+.program-hr {
+  width: 10%;
+  border: 1px solid #ec5242;
+  display: flex;
+  justify-content: center;
+  margin: 20px 45%;
+}
+
+.date {
+  padding: 20px 40px;
+  width: 100%;
+  background-color: rgba(256, 256, 256, 27%);
+  margin-bottom: 20px;
+  color: #ec5242;
+  font-size: 14px;
+  display: flex;
+  align-items: center;
+}
+
+.sub-text {
+  color: #fff;
+  padding: 10px 10px;
+  margin-left: 10px;
+  font-size: 11px;
+}
+
+.icon-hover:hover {
+  border: 2px solid #fff;
+}
+
+.icon {
+  width: 90px;
+  height: auto;
+  margin-right: 30px;
+}
+
+.partner {
+  width: 100%;
+}
+
+.speakers {
+  padding: 10px 20px;
+}
+
+.speaker1-2 {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+}
+
+.speaker6 {
+  display: none;
+  flex-direction: row;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.speaker3 {
+  display: none;
+  flex-direction: row;
+  margin-top: 20px;
+}
+
+.speaker5 {
+  display: none;
+  flex-direction: row;
+  margin-top: 20px;
+}
+
+.speaker4 {
+  display: none;
+  flex-direction: row;
+  margin-top: 20px;
+}
+
+.speaker6.active {
+  display: flex;
+}
+
+.speaker5.active {
+  display: flex;
+}
+
+.speaker4.active {
+  display: flex;
+}
+
+.speaker3.active {
+  display: flex;
+}
+
+.feacher-speakers {
+  font-size: 26px;
+  color: #0b0031;
+  text-align: center;
+  margin-top: 50px;
+}
+
+.speaker-hr1 {
+  width: 10%;
+  border: 1px solid #ec5242;
+  margin: 10px 45% 50px 45%;
+}
+
+.img-div {
+  width: 90px;
+  height: 80px;
+  background-image: url(../Images/pattern.jpg);
+  margin-right: 40px;
+  padding: 0;
+}
+
+.speaker-img {
+  width: 100px;
+  height: auto;
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.s-name {
+  color: #0b0031;
+  margin-top: 10px;
+}
+
+.s-position {
+  color: red;
+  font-size: 10px;
+}
+
+.s-info {
+  font-size: 12px;
+}
```

Comment:
- [ ] Kindly increase font size because it is small and it is difficult to read and increase also in program section.